SAMtags.tex

```diff
@@ -60,7 +60,8 @@ \section{Standard tags}
   {\tt BQ} & Z & Offset to base alignment quality (BAQ) \\
   {\tt BZ} & Z & Phred quality of the unique molecular barcode bases in the {\tt OX} tag \\
   {\tt CC} & Z & Reference name of the next hit \\
-  {\tt CM} & i & Edit distance between the color sequence and the color reference (see also {\tt NM}) \\
+  {\tt CG} & B,I & Intended to store the real {\sf CIGAR} if it contains $>$65535 operations\\
```

Comment:
I don't like "Intended" here as the specification isn't about what we'd like, but about what must happen.

How about "BAM only: stores the real {\sf CIGAR} field if it contains $>65535$ operations".